src/main/java/seedu/address/logic/parser/DateParserUtil.java

```diff
@@ -22,10 +22,13 @@
  * Helper class for {@link ParserUtil#parseDate(String)}.
  */
 public class DateParserUtil {
-    public static final String MESSAGE_CONSTRAINTS = "Date should contains a properly formatted date "
-            + "or a recognised natural date language.\n"
-            + "An example of a properly formatted date includes (but not limited to): 02 Aug 2020\n"
-            + "An example of a natural date language is 'today' or 'wednesday'(any day of the week)";
+    public static final String MESSAGE_CONSTRAINTS = "The Date input must adhere to our formatting conventions "
+            + "or adopt a recognised natural date language.\n"
+            + "An example of a properly formatted date includes (but is not limited to): 02 Aug 2020\n"
+            + "An example of a natural date language is 'today' or 'wednesday'(any day of the week)"
+            + "Please visit our user guide at "
+            + "https://ay2021s1-cs2103t-t12-4.github.io/tp/UserGuide.html#faq"
+            + " for more date/time variants.";
```

Comment:
Yep this looks much better! 👍
But is it abit long? Maybe can shorten the last part to refer to `help` instead?